apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -607,13 +607,13 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
                 <ColorPicker
                   defaultValue={eventTypeColorState.darkEventTypeColor}
                   onChange={(value) => {
+                    const newVal = {
+                      ...eventTypeColorState,
+                      darkEventTypeColor: value,
+                    };
+                    formMethods.setValue("eventTypeColor", newVal, { shouldDirty: true });
                     if (checkWCAGContrastColor("#101010", value)) {
-                      const newVal = {
-                        ...eventTypeColorState,
-                        darkEventTypeColor: value,
-                      };
                       setDarkModeError(false);
-                      formMethods.setValue("eventTypeColor", newVal, { shouldDirty: true });
                       setEventTypeColorState(newVal);
```

Comment:
```suggestion
```
Shouldn't this also be outside the if-else block?

Reply:
Yeah, I missed it